### DISCUSSION TITLE
im using pebblehost and i keep getting this error any help would be really appreciated

### DISCUSSION THREAD

**Discussion by UrafikiTheFluff** [jagrosh/MusicBot#1778] [Questions]
this is what console in pebblehost says and then closes out the server 
11-30 08:26:34 [PebbleHost Daemon]: Status: Image is up to date for pebblehost/pebble-bot:v20 
11-30 08:26:34 [PebbleHost Daemon]: Finished pulling container image
11-30 08:26:35 [PebbleHost Loader]: ----------------------------------------------
11-30 08:26:35 [PebbleHost Loader]: PebbleHost Bot Loader now starting...
11-30 08:26:35 [PebbleHost Loader]: Running loader version Bot-1.2.1
11-30 08:26:35 [PebbleHost Daemon]: Server marked as running
11-30 08:26:35 [PebbleHost Loader]: Checking for Configured Reverse Proxy...
11-30 08:26:35 [PebbleHost Loader]: Attempting to boot with the following settings: SERVER ID: 885515, MEMORY: 1024M
11-30 08:26:35 [PebbleHost Loader]: Found package.json
11-30 08:26:35 [PebbleHost Loader]: Node version v16.17.0
11-30 08:26:35 [PebbleHost Loader]: Updating NPM packages (command output is in logs/2024-11-30_08-26-35.txt)...
11-30 08:26:36 [PebbleHost Loader]: Checking if 'npm start' command is set up in package.json
11-30 08:26:36 [PebbleHost Loader]: Didn't find a start command - will use the main file bot.js
11-30 08:26:36 [PebbleHost Loader]: ----------------------------------------------
11-30 08:26:36 Hello World! The example application is currently running, you will need to upload your own bot to run.
11-30 08:26:36 [PebbleHost Daemon]: Server marked as offline
11-30 08:26:36 [PebbleHost Daemon]: ---------- Detected server process in a crashed state! ----------
11-30 08:26:36 [PebbleHost Daemon]: Exit code: 0
11-30 08:26:36 [PebbleHost Daemon]: Out of memory: false
11-30 08:26:36 [PebbleHost Daemon]: Aborting automatic restart, last crash occurred less than 600 seconds ago.

**Comment by vandevsysX64** [comment]
it needs jre not node js
